<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Harishyam" failures="3" tests="13" name="parallel.ParallelLoginRun" time="184.991" errors="0" timestamp="2024-12-27T04:58:50 IST" skipped="0">
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="12.263"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="11.056"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="4.617">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at parallel.LoginStepDef.page_title_should_be(LoginStepDef.java:60)
at ✽.Page Title should be "Dashboard"(classpath:parallel/login.feature:36)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="17.577"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="17.452"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="4.730">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at parallel.LoginStepDef.page_title_should_be(LoginStepDef.java:60)
at ✽.Page Title should be "Dashboard"(classpath:parallel/login.feature:36)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="18.102"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="17.908"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="23.850"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="17.036"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="4.697">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at parallel.LoginStepDef.page_title_should_be(LoginStepDef.java:60)
at ✽.Page Title should be "Dashboard"(classpath:parallel/login.feature:36)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="17.886"/>
  <testcase classname="parallel.ParallelLoginRun" name="runScenario" time="17.817"/>
</testsuite> <!-- parallel.ParallelLoginRun -->
